src/main/java/uk/co/ramp/covid/simulation/place/Place.java

```diff
@@ -111,10 +111,25 @@ protected double getTransConstant() {
         return pairs;
     }
 
+    // rng.nextBinomial() expects an integer - this method handles >Integer.MAX_VALUE trials 
```

Comment:
Should we add a comment that: Bin(n + m, p) = Bin(n,p) + Bin(m,p)? Just to make it clear why this function looks like it does?